applications/annotate_points.md

```diff
@@ -0,0 +1,383 @@
+
+```python
+stack = imread(im_path)
+```
+
+We can then start the viewer Note that we use the `napari.gui_qt()` context manager to start and manage Qt event loop. We must provide the `contrast_limits` and set `is_pyramid=False` to prevent the image stack from being evaluated (i.e., loaded) upon layer creation. All of the following GUI code should be within the `napari.gui_qt()` context manager.
+
+```python
+with napari.gui_qt():
+    viewer = napari.view_image(stack, contrast_limits=[0, 256])
+```
+
+## Annotating with points
+We will annotate the features of interest using points in a napari Points layer. Each feature will be given a different label so that we can track them across frames. To achieve this, we will store the label in the `Points.properties` property in the 'label' key. We will instantiate the `Points` layer without any points. Therefore, we will initialize `Points.properties` with the key we will use and an empty array (`properties={'label': np.empty(0)}`). We also initialize the layer with the valid label names using the `default_properties` kwarg (`default_properties = {'label': np.array(labels)}`).
+
+```python
+properties = {'label': np.array(labels)}
+```
+
+We then add the points layer to the viewer using the `viewer.add_points()` method. As discussed above, we will  To visualize the feature each points represent, we set the edge color as a color cycle mapped to the `label` property (`edge_color='label'`). 
+
+```python
+points_layer = viewer.add_points(
+    properties=properties,
+    edge_color='label',
+    edge_color_cycle=COLOR_CYCLE,
+    symbol='o',
+    face_color='transparent',
+    edge_width=8,
+    size=12
+)
+```
+
+Note that we set the `edge_color_cycle` to `COLOR_CYCLE`. You can define your own color cycle as a list of colors. The colors can be defined as hex strings, standard color names or RGBA arrays. For example, the [category10 color pallete](https://github.com/d3/d3-3.x-api-reference/blob/master/Ordinal-Scales.md#category10) would be:
+
+```python
+COLOR_CYCLE = [
+	'#1f77b4',
+	'#ff7f0e',
+	'#2ca02c',
+	'#d62728',
+	'#9467bd',
+	'#8c564b',
+	'#e377c2',
+	'#7f7f7f',
+	'#bcbd22',
+	'#17becf'
+]
+```
+
+Finally, we set the edge color to a color cycle:
+
+```python
+	points_layer.edge_color_mode = 'cycle'
+```
+
+## Adding a UI for selecting points
+Next, we will use magicgui to add a dropdown menu for selecting which point we would like to add. [magicgui](https://github.com/napari/magicgui) is a library for building GUIs from functions and works by applying function decorators. To make the a dropdown menu populated with the valid point labels, we simply define the function with the label as an input argument and then decorate it with the `magicgui()` decorator, passing the labels choice as `label={'choices': labels}` (recall that labels was passed to `point_annotator() as a list of the allowable labels).
```

Comment:
maybe say `[magicgui](https://github.com/napari/magicgui) is a library for building GUIs from functions from the napari team`. I think it matters that it is made by @tlambert03 and us in terms of signaling to users that it is something they can trust and that we will continue to support

Reply:
Sounds good!

Reply:
Fixed!